mediathread/settings_shared.py

```diff
@@ -219,8 +219,10 @@ def default_url_processor(url, label=None, request=None):
 if 'test' in sys.argv or \
    'jenkins' in sys.argv or \
    'integrationserver' in sys.argv:
-    DEBUG = False
+    DEBUG = True
```

Comment:
Why switch this to `True`?

Reply:
So when I get 500 errors in cypress, the error is displayed fully.